### PR TITLE
edited for grammatical clarity.

### PR DESCRIPTION
**Pull request by gyehuda**
If you have suggestions to help improve the clarity of this documentation, please suggest them. Thanks!

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: README.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
 
-AppDevKit is an iOS development library that provides foundational and developer everyday required features for their iOS app development. It has been used by Yahoo’s iOS app production for 3 years, and future outsourcing apps will also be using AppDevKit. The stability and scalability has been verified on these production apps. It makes difficult development tasks easier and has saved 30% development time in real case. It also covers incompatibility issues caused by different iOS platforms.
+AppDevKit is an iOS development library that provides fundamental and usefull features to help your iOS app development. This kit has used by Yahoo’s iOS app development team in Taiwan for the past 3 years, and we plan on having future outsourced apps use AppDevKit too. Our use of this code in many production environments helped improve its stability and scalability. We find thes libraries make difficult development tasks easier and has saved us up to 30% development time in some cases. The kit also helps address some incompatibility issues found in different iOS versionss.
```
Comment (nbroslawsky) [inline README.md]: "usefull" is misspelled.

Comment (nbroslawsky) [inline README.md]: "We find thes libraries". Might want to fix that to be "these"

File: README.md [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
 
-It has 5 major parts that include command, user interfaces, animate, image and list view support libraries. Please leverage AppDevKit in your iOS project or join our development group of AppDevKit. We will maintain this project for you.
+AppDevKit has 5 major parts. These are: command, user interfaces, animation, image, and list view support libraries. Please feel welcome to use AppDevKit in your iOS projects and help contribute improvements to too.
 
-If you have any idea to improve this project, please feel free to contact with me (**cfsung@yahoo-inc.com**) and core team (**app-dev-kit@yahoo-inc.com**) or send **Pull Request** to us. Thank you. 
+If you have suggestions about how to improve this project, please feel free to contact (**cfsung@yahoo-inc.com**) and core development team (**app-dev-kit@yahoo-inc.com**) or send **Pull Request** to this project. Thank you.
```
Comment (nbroslawsky) [inline README.md]: "send **a** pull request"

Comment (gyehuda) [inline README.md]: thanks -- all good corrections

**Comment by ihuxley** [comment]
Consider removing the link to the presentation that is linked to at http://www.slideshare.net/anistarsung/appdevkit-for-ios-development, it also has a few issues and I think the git readme already does a nice job of describing the functionality.

Also I suggest removing the claim that this library has saved 30% in development time - that is a very bold claim that is very difficult to demonstrate to be accurate in an objective way.

**Comment by anistarsung** [comment]
I closed and moved this PR to https://github.com/yahoo/AppDevKit/pull/16. Because it out-of-date and can't be merged.

**Pull request closed without merging**